test/functional/feature_taproot.py

```diff
@@ -218,6 +222,20 @@ def default_controlblock(ctx):
     """Default expression for "controlblock": combine leafversion, negflag, pubkey_internal, merklebranch."""
     return bytes([get(ctx, "leafversion") + get(ctx, "negflag")]) + get(ctx, "pubkey_internal") + get(ctx, "merklebranch")
 
+def default_scriptcode_suffix(ctx):
```

Comment:
Should it be named `default_scriptcode_wsh` to be explicit that it implements the Segwit v0 (and not legacy) logic with regard to `OP_CODESEPARATOR` serialization?

Reply:
It did, legacy skips `OP_CODESEPARATOR`s in the suffix (i.e. "upcoming `CODESEP`s") whereas Segwit v0 doesn't.